src/auro-calendar-cell.js

```diff
@@ -139,7 +187,12 @@ export class AuroCalendarCell extends LitElement {
     });
   }
 
-  getSlotName() {
+  /**
+   * Gets the name of the date slot.
+   * @private
+   * @returns {void}
+   */
+  getDateSlotName() {
```

Comment:
Let's use a slightly more semantic function name here. `get` would imply we are returning the name.

```suggestion
  setDateSlotName() {
```

Reply:
Method name updated